### PR TITLE
Added documentation about Insights tab (CCXDEV-1285)

### PR DESCRIPTION
**Pull request by mmuehlfeldRH**
This update adds documentation about the Insights tab on cloud.redhat.com to the "Support" section of the docs.

This doc enhancement is for OpenShift 4.5.

@openshift/team-documentation

### CODE REVIEW

**Comment by radekvokal** [comment]
Reviewed by CCX PM - ack

File: modules/displaying-potential-issues-of-your-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc
+
+[id="displaying-potential-issues-of-your-cluster_{context}"]
+= Displaying potential issues of your cluster
+
+This section describes how to display the Insights report the {cloud-redhat-com} has created.
```
Comment (janzeleny) [inline modules/displaying-potential-issues-of-your-cluster.adoc:8]: This translates to statement that OCM generates the Insights report for you which is factually wrong and might be confusing in the future when OCM is not the only UI we use. My choice of words would be something in lines of "Insights generates the report, OCM displays it".

File: modules/displaying-potential-issues-of-your-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+. Click to the cluster's name to display the details of the cluster.
+
+. Open the *Insights* tab of the cluster.
++
+Depending on the result, the tab displays one of the following:
++
+* *Your cluster passed all health checks*, if {cloud-redhat-com} did not identify any issues.
+
+* A list of issues {cloud-redhat-com} has detected, prioritized by risk.
```
Comment (janzeleny) [inline modules/displaying-potential-issues-of-your-cluster.adoc:30]: Same as above, OCM does not detect the issues, just shows the results.

File: modules/displaying-potential-issues-of-your-cluster.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+. Click the *Clusters* menu in the left pane.
+
+. Click to the cluster's name to display the details of the cluster.
+
+. Open the *Insights* tab of the cluster.
++
+Depending on the result, the tab displays one of the following:
++
+* *Your cluster passed all health checks*, if {cloud-redhat-com} did not identify any issues.
```
Comment (janzeleny) [inline modules/displaying-potential-issues-of-your-cluster.adoc:28]: Same as above, OCM does not detect the issues, just shows the results.

File: modules/displaying-potential-issues-of-your-cluster.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+* A list of issues {cloud-redhat-com} has detected, prioritized by risk.
+
+* *No health checks to display*, if {cloud-redhat-com} has not yet analyzed the cluster. The analysis starts shortly after the cluster has been installed and connected to the internet.
```
Comment (janzeleny) [inline modules/displaying-potential-issues-of-your-cluster.adoc:32]: Same as above, OCM does not detect the issues, just shows the results.

File: support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc [hunk 5]
```diff
@@ -0,0 +1,10 @@
+[id="using-insights-to-identify-issues-of-your-cluster"]
+= Using Insights to identify issues of your cluster
+include::modules/common-attributes.adoc[]
+:context: using-insights-to-identify-issues-of-your-cluster
+toc::[]
+
+{cloud-redhat-com} repeatedly analyzes the data Insights Operator sends. Users of {product-title} can display the report on the *Insights* tab of each cluster in {cloud-redhat-com}.
```
Comment (janzeleny) [inline support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc:7]: Same as above, OCM does not detect the issues, just shows the results.

**Review by janzeleny — CHANGES_REQUESTED** [review]

**Comment by janzeleny** [comment]
Thanks for addressing my comments - ack

**Comment by mmuehlfeldRH** [comment]
@openshift/team-documentation , can you please review the PR and merge it if everything is OK? Thanks.

File: modules/displaying-potential-issues-of-your-cluster.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+* Your cluster is registered in the {cloud-redhat-com}.
+* Remote health reporting is enabled, which is the default.
+* You are logged in to the {cloud-redhat-com}.
+
+.Procedure
+
+. Click the *Clusters* menu in the left pane.
+
+. Click to the cluster's name to display the details of the cluster.
```
Comment (ahardin-rh) [inline modules/displaying-potential-issues-of-your-cluster.adoc:22]: s/to the cluster's name/the cluster's name

File: support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc [hunk 7]
```diff
@@ -0,0 +1,10 @@
+[id="using-insights-to-identify-issues-of-your-cluster"]
+= Using Insights to identify issues of your cluster
```
Comment (ahardin-rh) [inline support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc:2]: s/of your/with your

**Comment by ahardin-rh** [comment]
@yapei Can you please provide a QE review? Thanks!

File: modules/displaying-potential-issues-with-your-cluster.adoc [hunk 8]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-of-your-cluster.adoc
+
+[id="displaying-potential-issues-of-your-cluster_{context}"]
+= Displaying potential issues of your cluster
```
Comment (ahardin-rh) [inline modules/displaying-potential-issues-with-your-cluster.adoc:6]: with your cluster

**Comment by yapei** [comment]
@ahardin-rh  OpenShift UI QE didn't test this Insights feature, maybe this is owned by Insights team?

**Comment by ahardin-rh** [comment]
@jneedle Can you or a member of your team provide QE review? Thanks!

**Comment by jneedle** [comment]
Let me see if I can find someone on that team to have a look.

File: modules/displaying-potential-issues-with-your-cluster.adoc (hunk 8 above)
Comment (mmuehlfeldRH) [reply modules/displaying-potential-issues-with-your-cluster.adoc]: Sorry. I missed this one. Fixed.

**Review by tisnik — APPROVED** [review]
LGTM @mmuehlfeldRH

**Review by bond95 — COMMENTED** [review]
Basically looks good, but could we say a little bit more about Remediate modal window and maybe slightly more describe each Total risk level. WDYT?

**Comment by mmuehlfeldRH** [comment]
@bond95 I added the risk levels to step 3 (second bullet), and updated the last paragraph in step 4.

**Review by bond95 — APPROVED** [review]
LGTM, thank you!

**Comment by ahardin-rh** [comment]
Thank you, all!

**Pull request merged**